courses/udacity_deep_learning/4_convolutions.ipynb

```diff
@@ -139,7 +139,7 @@
         "id": "5rhgjmROXu2O"
       },
       "source": [
-        "Let's build a small network with two convolutional layers, followed by one fully connected layer. Convolutional networks are more expensive computationally, so we'll limit its depth and number of fully connected nodes."
+        "Let's build a small network with two convolutional layers, followed by one fully connected layer. Convolutional networks are more expensive computationally, so we'll limit their depth and number of fully connected nodes."
```

Comment:
This should be "its", it refers to the small network that we are building in this cell.